operations/host_spawn.go

```diff
@@ -383,6 +382,7 @@ func hostConfigure() cli.Command {
 				}
 
 				if err := cmd.Run(ctx); err != nil {
+					grip.Infof("You may need to accept an email invite to receive access to %s.", projectRef.Repo)
```

Comment:
nit: can you change to "repo '%s'" ? so it's clear that the word being added is a repo (this message is likely for new hires so good to be explicit). We frequently wrap formatting with ticks so that it's clear that it's a quoted variable